expan/core/statistics.py

```diff
@@ -85,16 +110,28 @@ def delta(x, y, assume_normal=True, alpha=0.05, min_observations=20, nruns=10000
         if assume_normal:
             logger.info("The distribution of two samples is assumed normal. "
                         "Performing the sample difference distribution calculation.")
-            c_i = normal_sample_difference(x=_x, y=_y, percentiles=percentiles, relative=relative)
+            lots_of_info = normal_sample_weighted_difference(x_numerators=_x, y_numerators=_y, x_denominators=_x_denominators, y_denominators = _y_denominators, percentiles=percentiles, relative=relative)
```

Comment:
this name of the variable is not straightforward to understand. Maybe some more meaningful name?

Reply:
Good point :-)  It's quite similar to the delta_statistics. I might just call it `partial_delta_statistics`. I'll push something in a few minutes